src/routes/stop_monitoring.rs

```diff
@@ -76,6 +78,13 @@ fn create_monitored_stop_visit(
     }
 }
 
+fn get_line_ref<'a>(cnx: &Connection, model: &'a navitia_model::Model) -> &'a str {
+    let vj = &model.vehicle_journeys[cnx.dated_vj.vj_idx];
+    let route = model.routes.get(&vj.route_id).unwrap(); // should we have this unwrap ? do we consider that we can't have bad data ?
```

Comment:
what do you think about this @Tristramg ?

Reply:
A model can't be build without I'd coherency. get_corresponding allow you to get the line from the vj without using the route explicitly